angular-todo/playwright.config.ts

```diff
@@ -0,0 +1,78 @@
+import { defineConfig, devices } from '@playwright/test';
+
+const url = process.env.PLAYWRIGHT_TEST_BASE_URL || 'https://localhost:8443';
+
+export default defineConfig({
+  testDir: 'e2e',
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: 'html',
+  use: {
+    trace: 'on',
+    baseURL: `${url}`,
+    ignoreHTTPSErrors: true,
+    headless: !!process.env.CI,
+  },
+  webServer: [
+    {
+      command: 'npm run start:angular-todo',
+      url,
+      timeout: 120 * 1000,
+      reuseExistingServer: !process.env.CI,
+      cwd: '../',
+      env: {
+        API_URL: 'http://localhost:9443',
+        DEBUGGER_OFF: 'true',
+        DEVELOPMENT: 'false',
+        JOURNEY_LOGIN: 'Login',
+        JOURNEY_REGISTER: 'Registration',
+        PORT: '9443',
+        AM_URL: 'https://openam-sdks.forgeblocks.com/am',
+        REALM_PATH: 'alpha',
+        SCOPE: 'profile me.read email',
+        TIMEOUT: '3000',
+        WEB_OAUTH_CLIENT: 'CentralLoginOAuthClient-',
+        REST_OAUTH_CLIENT: 'RestOAuthClient',
+        REST_OAUTH_SECRET: 'newsecret121314!',
+        CENTRALIZED_LOGIN: 'false',
+      },
+      ignoreHTTPSErrors: true,
+    },
+  ],
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] },
+    },
+
+    /* Test against mobile viewports. */
+    // {
+    //   name: 'Mobile Chrome',
+    //   use: { ...devices['Pixel 5'] },
+    // },
+    // {
+    //   name: 'Mobile Safari',
+    //   use: { ...devices['iPhone 12'] },
+    // },
+
+    /* Test against branded browsers. */
+    // {
+    //   name: 'Microsoft Edge',
+    //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+    // },
+  ],
+
+  /* Run your local dev server before starting the tests */
+  // webServer: {
+  //   command: 'npm run start',
+  //   url: 'http://127.0.0.1:3000',
+  //   reuseExistingServer: !process.env.CI,
+  // },
```

Comment:
Do we need to keep all of this, or can we remove it?

Reply:
certainly can, keeping it just to share the available options.

Reply:
done